motoko/basic_bitcoin/src/basic_bitcoin/src/EcdsaApi.mo

```diff
@@ -0,0 +1,49 @@
+import Types "Types";
+
+module {
+  type ECDSAPublicKey = Types.ECDSAPublicKey;
+  type ECDSAPublicKeyReply = Types.ECDSAPublicKeyReply;
+  type SignWithECDSA = Types.SignWithECDSA;
+  type SignWithECDSAReply = Types.SignWithECDSAReply;
+
+  /// Actor definition to handle interactions with the ECDSA canister.
+  type EcdsaCanisterActor = actor {
+      ecdsa_public_key : ECDSAPublicKey -> async ECDSAPublicKeyReply;
+      sign_with_ecdsa : SignWithECDSA -> async SignWithECDSAReply;
+  };
+
+  let KEY_NAME : Text = "test";
```

Comment:
The key name must also be updated before we release the code, right?

Reply:
[Islam confirmed.](https://github.com/dfinity/examples/pull/280#discussion_r916986738)